part1/color.rb

```diff
@@ -0,0 +1,26 @@
+require 'pry'
+
+class Color
+  attr_reader :r, :g, :b
+  def initialize(r, g, b)
+    @r = r
+    @g = g
+    @b = b
+  end
+
+  def brightness_index
+    (299*r + 587*g + 114*b) / 1000
```

Comment:
always try to avoid magic number in you codebase. Use constants instead:
`SOME_NUMBER = 299`